api/lib/controllers/invite.js

```diff
@@ -0,0 +1,111 @@
+const Invite = require("../models/invite");
+const User = require("../models/user");
+const authUtils = require("../util/authorization");
+
+module.exports = {
+  create: async (req, res) => {
+    const { group_id, invitee } = req.body;
+
+    await authUtils.checkOwner(group_id, "groups", req.credentials);
+
+    const inviteeUser = await User.findOne({ _id: invitee });
+
+    if (!inviteeUser) {
+      return res.status(400).send({ message: "Invitee not found" });
+    }
+
+    const existingInvite = await Invite.findOne({
+      group_id,
+      invitee,
+      status: "open",
+    });
```

Comment:
does this just know what group_id is from the variable name?

Reply:
Yeah, since the variable name matches it is equivalent to `group_id: group_id`.